internal/utils/streamframe_interval.go

```diff
@@ -0,0 +1,45 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/quic-go/quic-go/internal/protocol"
+)
+
+// ByteInterval is an interval from one ByteCount to the other
+type ByteInterval struct {
```

Comment:
I assume this implementing an interface needed by the tree. Can you add an interface assertion?